examples/sentence-transformers-training/nli/readme.md

```diff
@@ -0,0 +1,64 @@
+# Natural Language Inference
+
+Given two sentence (premise and hypothesis), Natural Language Inference (NLI) is the task of deciding if the premise entails the hypothesis, if they are contradiction, or if they are neutral. Commonly used NLI dataset are [SNLI](https://huggingface.co/datasets/stanfordnlp/snli) and [MultiNLI](https://huggingface.co/datasets/nyu-mll/multi_nli). 
+
+[Conneau et al.](https://arxiv.org/abs/1705.02364) showed that NLI data can be quite useful when training Sentence Embedding methods. We also found this in our [Sentence-BERT-Paper](https://arxiv.org/abs/1908.10084) and often use NLI as a first fine-tuning step for sentence embedding methods.
+
+
+## Requirements
+
+First, you should install -
+```bash
+pip install -U sentence-transformers
+pip install git+https://github.com/huggingface/optimum-habana.git
+```
+
+
+## Usage
+
+To pre-training on NLI dataset you can do following steps -
+
```

Comment:
Can you explicitly add the fine-tuning instruction for this task too?

Reply:
why we need to add additional fine-tuning part and each example just for each intention for simplicity? This example is directly copied from sentence transformer example. We need keep consistent. Thanks.

Reply:
To make the readmes consistent across all task, we should provide pre-training and fine-tuning information across all supported tasks.